src/js/actions/DialogActions.js

```diff
@@ -0,0 +1,66 @@
+var AppDispatcher = require("../AppDispatcher");
+var DialogEvents = require("../events/DialogEvents");
+
+var DialogActions = {
+  alert: function (message) {
+    const dialogId = Symbol(message);
+    AppDispatcher.dispatchNext({
+      actionType: DialogEvents.ALERT,
+      message: message,
+      dialogId: dialogId
+    });
+    return dialogId;
+  },
+  alertClose: function (dialogId) {
+    AppDispatcher.dispatchNext({
+      actionType: DialogEvents.ALERT_CLOSE,
+      dialogId: dialogId
+    });
+  },
+  confirm: function (message) {
+    const dialogId = Symbol(message);
+    AppDispatcher.dispatchNext({
+      actionType: DialogEvents.CONFIRM,
```

Comment:
Great! Then I will submit a PR that 

- [ ] adds the `_SHOW` suffixes
- [ ] renames `_CLOSE` to `_DISMISS`
- [ ] replaces all occurrences of `destroy` with `dismiss`